### PR TITLE
Added samples for functions which are not deprecated in ch.tutteli.at…

### PR DESCRIPTION
**Pull request by VarunEnishetty**
…rium.api.infix.en_GB.samples.BigDecimalExpectationSamples

______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/main/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt [hunk 1]
```diff
@@ -28,4 +29,44 @@ class BigDecimalExpectationSamples {
             expect(null as BigDecimal?) notToEqual null
         }
     }
+
+    @Test
+    fun toEqualNumerically(){
```
Comment (robstoll) [inline apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt:34]: please add the link to this and the other function in the KDOC (via `@sample`) of the corresponding functions in
apis/infix/atrium-api-infix/src/jvmMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/bigDecimalExpectations.kt 

See toEqual

Comment (robstoll) [inline apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt:34]: please add the link to this and the other function in the KDOC (via `@sample`) of the corresponding functions in
apis/infix/atrium-api-infix/src/jvmMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/bigDecimalExpectations.kt 

See toEqual

**Review by robstoll — CHANGES_REQUESTED** [review]

File: apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt (hunk 1 above)
Comment (VarunEnishetty) [reply apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt:34]: @robstoll Sorry, I missed to check-in this file. I added new commit. Please review it.

Comment (robstoll) [reply apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt:34]: No worries, LGTM now :+1:, let's see what CI means

Comment (robstoll) [reply apis/infix/atrium-api-infix/src/jvmTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/BigDecimalExpectationSamples.kt:34]: No worries, LGTM now :+1:, let's see what CI means

**Review by robstoll — APPROVED** [review]

**Comment by robstoll** [comment]
@VarunEnishetty thanks for your first contribution to Atrium :+1: 

How about another one? 

(and don't forget to Star Atrium if you like it)

**Comment by VarunEnishetty** [comment]
@robstoll , sure would love to even pickup any dev tasks as well

**Comment by robstoll** [comment]
@VarunEnishetty nice, take a look at the open issues where we could use your help https://github.com/robstoll/atrium/issues?q=is%3Aopen+label%3A%22help+wanted%22+no%3Aassignee

**Pull request merged**